contracts/scripts/compileSol.sh

```diff
@@ -12,7 +12,7 @@ mkdir -p ./compiled/abis
 
 # Copy the Semaphore contracts from the submodule into solidity/
 
-npx etherlime compile --solcVersion=native --buildDirectory=compiled --workingDirectory=sol --exportAbi --runs 200
+npx etherlime compile --solcVersion=0.5.16 --buildDirectory=compiled --workingDirectory=sol --exportAbi --runs 200
```

Comment:
```suggestion
if [[ $* == *--native* ]]; then
    npx etherlime compile --solcVersion=native --buildDirectory=compiled --workingDirectory=sol --exportAbi --runs 200
else
    npx etherlime compile --solcVersion=0.5.16 --buildDirectory=compiled --workingDirectory=sol --exportAbi --runs 200
fi
```

I suggest giving the developer the option to use the `native` flag as the solcjs compiler is slower than the solc binary.

Reply:
Ah, sure! I didn't know about that.
Though I'd like to write it differently: 81f85f1af667cb14bfacd058ffd1b8a09e219dad